src/ui/src/wds/WdsDropdownMenu.vue

```diff
@@ -49,6 +55,9 @@
 			</button>
 		</template>
 
+		<template v-else-if="hasNoResults">
+			<div class="WdsDropdownMenu__no-results">No results</div>
+		</template>
```

Comment:
`<template>` is not needed
```suggestion
	  <div v-else-if="hasNoResults" class="WdsDropdownMenu__no-results">No results</div>
```